src/res_tracker.h

```diff
@@ -42,11 +42,14 @@ class ResTracker {
   /// decay).
   void Modify();
 
-  /// Should be called when a resource's package gets modified
-  void Package();
+  /// Should be called when a resource's package gets modified. If the resource
+  /// was just created from a parent resource, the parent should be passed in.
+  /// If the resource is just being repackaged (e.g. to unpackaged), the parent
+  /// should be NULL.
+  void Package(ResTracker* parent = NULL);
 
  private:
-  void Record();
+  void Record(bool no_bump = false);
```

Comment:
What are your thoughts on flipping the logic on this?
```suggestion
  void Record(bool bumpId = true);
```

My sense it that there is a little extra cognitive load to thinking through what it means to set a variable with a negative name to true....